tests/test_admin.py

```diff
@@ -0,0 +1,70 @@
+from pathlib import Path
+
+from django.core.files.base import File
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+
+import pytest
+from pyquery import PyQuery as pq
+from zds_client.client import ClientError
+
+from zgw_consumers.constants import APITypes
+from zgw_consumers.models import Service
+from zgw_consumers.test import mock_service_oas_get
+
+API_ROOT = "https://example.com/api/v1/"
+OAS_PATH = Path(__file__).parent / "schemas/ztc.yaml"
+TESTS_DIR = Path(__file__).parent
+
+
+@pytest.mark.django_db
+def test_listzaaktypen_mixin(settings, admin_client, requests_mock):
+    """
+    Assert that an error message is displayed in the admin if ClientError is raised
+    """
+    settings.ZGW_CONSUMERS_TEST_SCHEMA_DIRS = [TESTS_DIR / "schemas"]
+    mock_service_oas_get(requests_mock, API_ROOT, "ztc")
+
+    # define errors
+    error_cause = ClientError()
+    error_cause.args = [{"status": 400, "detail": "some other error"}]
+    err = ClientError()
+    err.args = [
+        {
+            "title": "Je hebt geen toestemming om deze actie uit te voeren.",
+            "status": 403,
+            "detail": "Client identifier bestaat niet",
+        }
+    ]
+    err.__cause__ = error_cause
```

Comment:
this doesn't make a lot of sense, you are creating an exception to be thrown by the  mocking library instead of mocking the actual request and returning the error response as the api would return:

Correct mocking would be:

```py
requests_mock.get(
    endpoint,
    status_code=403,
    json={
        "title": "Je hebt geen toestemming om deze actie uit te voeren.",
        "status": 403,
        "detail": "Client identifier bestaat niet",
    },
)
```

See https://requests-mock.readthedocs.io/en/latest/response.html#registering-responses

So I'd also like to see a test for an HTTP 5xx error.